### PR TITLE
Bring back logging arguments

### PR DESCRIPTION
**Pull request by przybylski**
Needed for external SD support

### CODE REVIEW

File: src/com/owncloud/android/lib/common/utils/Log_OC.java [hunk 1]
```diff
@@ -103,6 +103,29 @@ public static void startLogging() {
         } 
     }
 
+    synchronized public static void stopLogging() {
+        try {
+			if (mBuf != null)
```
Comment (tobiasKaminsky) [inline src/com/owncloud/android/lib/common/utils/Log_OC.java]: indentation

**Comment by AndyScherzinger** [comment]
I fixed the identation, one thing I realsized looking through the code, shouldn't we replace the system out println statements with standard Android log statements?

**Comment by przybylski** [comment]
@AndyScherzinger yes we should, but where is those used ? Whenever I see System.out.print* function call i replace it

**Comment by AndyScherzinger** [comment]
https://github.com/nextcloud/android-library/pull/5/files#diff-0eb21eac7b9b2e2b07a7f7a7945ce689R101
https://github.com/nextcloud/android-library/pull/5/files#diff-0eb21eac7b9b2e2b07a7f7a7945ce689R107

**Comment by przybylski** [comment]
Waiting for review :)

**Comment by AndyScherzinger** [comment]
:+1:

**Comment by AndyScherzinger** [comment]
Can you have an look @tobiasKaminsky ?

**Comment by przybylski** [comment]
I wonder if I put myself a thumbs up will it work, since Andy is a coauthor now

**Comment by przybylski** [comment]
👍

**Comment by AndyScherzinger** [comment]
it probably only care about who actually opened the PR, besides the LGTM check seems to not work properly at the moment

**Comment by tobiasKaminsky** [comment]
LGTM

**Pull request merged**